tests/config_test.go

```diff
@@ -240,7 +254,18 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			BeforeEach(func() {
 				for i := 0; i < secretsCnt; i++ {
 					name := "secret-" + uuid.NewString()
-					tests.CreateSecret(name, testsuite.GetTestNamespace(nil), map[string]string{"option": "value"})
+					data := map[string]string{
+						"option": "value",
+					}
+					virtCli := kubevirt.Client()
+
+					_, err := virtCli.CoreV1().Secrets(testsuite.GetTestNamespace(nil)).Create(context.Background(), &k8sv1.Secret{
+						ObjectMeta: metav1.ObjectMeta{Name: name},
+						StringData: data,
+					}, metav1.CreateOptions{})
+					if !errors.IsAlreadyExists(err) {
+						util2.PanicOnError(err)
```

Comment:
@sarthaksarthak9 please avoid to panic and include an assertion as it is reported in the https://github.com/kubevirt/kubevirt/issues/11437 description